rest_framework/fields.py

```diff
@@ -778,6 +778,8 @@ def to_internal_value(self, data):
         return data
 
     def to_representation(self, value):
+        if value is None:
+            return None
```

Comment:
OK. Good link. I see it. 

I'll have a look later. 

(Do you have any thoughts on keeping/dropping Django 1.7 here? Happy to keep it but not sure how, since the `if ...VERSION` fixes didn't work...?)